pages/index.js

```diff
@@ -1,13 +1,26 @@
-import Link from 'next/link';
+import Button from "@/components/form/Button";
 
 export default function Home() {
   return (
     <>
       <h1 className={styles.header}>Welcome to Amcovad</h1>
+
+      <h2>Solid Button</h2>
+      <Button>Sign in</Button>
+
+      <Button color="secondary">try it now</Button>
+
+      <h2>Outline Button</h2>
+
+      <Button outline>Sign in</Button>
+
+      <Button outline color="secondary">
+        try it now
```

Comment:
Even though you have made this to be uppercase, it is best practice to write this in the correct case. This is useful in case we want to remove the uppercase property in the future.

```suggestion
        Try it now
```